fcl/icarus_SH_queue.fcl.in

```diff
@@ -59,6 +59,7 @@ outputs: {
       ]
       streamName: "out"
    }
+
```

Comment:
This change just adds whitespace. Let’s not make commits that do that except in a PR that only addresses formatting.